src/client/src/services/trades/types.ts

```diff
@@ -49,6 +49,6 @@ export interface CreditTrade extends Trade {
 }
 
 export interface Trade {
-  tradeId: string
+  tradeId: bigint
```

Comment:
have to follow through the workings, but can we use the orig Trade from `TradingGateway.ts` now?
then we do not need to use TradeRaw with its weird capitalisation ...

Reply:
not sure why we need this loose type def when we have clearly defined types for FX and Credit trades ..